package.json

```diff
@@ -5,14 +5,17 @@
   "private": true,
   "dependencies": {
     "@codemirror/theme-one-dark": "^6.1.0",
+    "@headlessui/react": "^1.7.7",
+    "@heroicons/react": "^2.0.13",
```

Comment:
Since we're not using very many icons, we could just copy the code for the few that we need from the site and remove this dependency. https://heroicons.com/

Reply:
you are right